plugins/modules/web_infrastructure/django_manage.py

```diff
@@ -12,24 +13,36 @@
 DOCUMENTATION = '''
 ---
 module: django_manage
-short_description: Manages a Django application.
+short_description: Manages a Django application
 description:
-    - Manages a Django application using the C(manage.py) application frontend to C(django-admin). With the
-      C(virtualenv) parameter, all management commands will be executed by the given C(virtualenv) installation.
+  - Manages a Django application using the C(manage.py) application frontend to C(django-admin). With the
+    I(virtualenv) parameter, all management commands will be executed by the given C(virtualenv) installation.
 options:
   command:
     description:
-      - The name of the Django management command to run. Built in commands are C(cleanup), C(collectstatic),
-        C(flush), C(loaddata), C(migrate), C(syncdb), C(test), and C(validate).
-      - Other commands can be entered, but will fail if they're unknown to Django.  Other commands that may
+      - The name of the Django management command to run. The commands listed below are built in this module and have some basic parameter validation.
+      - >
+        C(cleanup) - clean up old data from the database (deprecated in Django 1.5). This parameter will be
+        deprecated in community.general 9.0.0. Use C(clearsessions) instead.
```

Comment:
I think you mean
```suggestion
        removed in community.general 9.0.0. Use C(clearsessions) instead.
```
It's deprecated with this PR. Same below.